app/src/test/java/org/simple/clinic/selectcountry/SelectCountryEffectHandlerTest.kt

```diff
@@ -0,0 +1,65 @@
+package org.simple.clinic.selectcountry
+
+import com.nhaarman.mockito_kotlin.doReturn
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import com.nhaarman.mockito_kotlin.verifyNoMoreInteractions
+import com.nhaarman.mockito_kotlin.whenever
+import io.reactivex.Single
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.simple.clinic.appconfig.AppConfigRepository
+import org.simple.clinic.appconfig.Country
+import org.simple.clinic.appconfig.ManifestFetchResult
+import org.simple.clinic.appconfig.ManifestFetchSucceeded
+import org.simple.clinic.mobius.EffectHandlerTestCase
+import org.simple.clinic.util.scheduler.TrampolineSchedulersProvider
+import java.net.URI
+
+class SelectCountryEffectHandlerTest {
+
+  private val repository = mock<AppConfigRepository>()
+
+  private val effectHandler = SelectCountryEffectHandler.create(repository, TrampolineSchedulersProvider())
+  private val testCase = EffectHandlerTestCase(effectHandler)
+
+  @Before
+  fun setUp() {
+    whenever(repository.fetchAppManifest()) doReturn Single.never<ManifestFetchResult>()
```

Comment:
Is this necessary? Is it not okay to just stub out calls that are being exercised in the appropriate test case?

Reply:
Done.